src/gazebo2rviz/gazebo2moveit_collision_node.py

```diff
@@ -31,12 +36,17 @@ def on_model_states_msg(model_states_msg):
     lastUpdateTime = rospy.get_rostime()
 
     for (model_idx, modelinstance_name) in enumerate(model_states_msg.name):
+        # Get the model name
```

Comment:
this comment seems redundant to me, the command is clear enough. Same reason for comment as above